sdk/src/test/java/com/deploygate/sdk/CustomAttributesInterfaceTest.java

```diff
@@ -0,0 +1,68 @@
+package com.deploygate.sdk;
+
+import androidx.annotation.NonNull;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * This test class will make sure all *public* interfaces are defined as expected
+ */
+@RunWith(AndroidJUnit4.class)
+public class CustomAttributesInterfaceTest {
+
+  @NonNull
+  CustomAttributes attributes;
+
+  @Before
+  public void setUp() {
+    attributes = new CustomAttributes();
+  }
+
+  @Test
+  public void putString() {
+    attributes.putString("key", "value");
```

Comment:
The signature is consist of *the name*, *the arguments*, *the modifiers* and **the return type**. 

It may be difficult to check the modifiers, but it's easy to check if the returned value is Boolean. Could you please add assertions?